packages/create/src/index.ts

```diff
@@ -55,6 +64,11 @@ export interface CreateOptions {
    * Use recommended bootstrap method to discovery and connect to new nodes.
    */
   defaultBootstrap?: boolean;
+
+  /**
+   * Node requirements to setup discovery with
+   */
+  nodeRequirements?: Partial<NodeCapabilityCount>;
```

Comment:
Not sure this is necessary, if a dev wants to fine tune their discovery experience, then they should not use default bootstrap.